scripts/downgrade.sh

```diff
@@ -186,6 +186,8 @@ downgrade_bootstrap () {
     kubeconfig=/etc/kubernetes/admin.conf
   $SALT salt-run state.sls metalk8s.orchestrate.deploy_node saltenv="$SALTENV" \
     pillar="{'orchestrate': {'node_name': '$bootstrap_id'}}"
+   _check_salt_master
+  $SALT salt-run saltutil.sync_all saltenv="metalk8s-$DESTINATION_VERSION"
```

Comment:
Don't do it here. You need to do it after the `metalk8s.roles.bootstrap` so move this right after `_check_salt_master` already present above